google/cloud/spanner_dbapi/cursor.py

```diff
@@ -207,9 +214,31 @@ def fetchone(self):
         self._raise_if_closed()
 
         try:
-            return next(self)
+            res = next(self)
+            self._checksum.consume_result(res)
+            return res
         except StopIteration:
-            return None
+            return
+        except Aborted:
+            self.connection.retry_transaction()
+            return self.fetchone()
+
+    def fetchall(self):
+        """Fetch all (remaining) rows of a query result, returning them as
+        a sequence of sequences.
+        """
+        self._raise_if_closed()
+
+        res = []
+        try:
+            for row in self.__iter__():
```

Comment:
```suggestion
            for row in self:
```

Nit: don't think you need the explicit `__iter__`.

Reply:
That's true